test/test_bort.F90

```diff
@@ -25,12 +28,57 @@ program test_bort
   print *, 'Testing ', sub_name, ' case ', test_case
 
   ! Run the test for the subroutine and test case.
-  if (sub_name .eq. 'bort') then
+  if (sub_name .eq. 'adn30') then
+     if (test_case .eq. '1') then
+        char_short = adn30(0, 5)
+     elseif (test_case .eq. '2') then
+        char_30 = adn30(0, 9)
+     endif
+  elseif (sub_name .eq. 'bort') then
      if (test_case .eq. '1') then
         call bort('goodbye!')
-     else if (test_case .eq. '2') then
+     endif
+  elseif (sub_name .eq. 'bort2') then
+     if (test_case .eq. '1') then
         call bort2('goodbye!', 'goodbye again!')
      endif
+  elseif (sub_name .eq. 'bvers') then
+     if (test_case .eq. '1') then
+        call bvers(char_short)
+     endif
+  elseif (sub_name .eq. 'cmpmsg') then
+     if (test_case .eq. '1') then
+        call cmpmsg('W')
+     endif
+  elseif (sub_name .eq. 'writdx') then
+     if (test_case .eq. '1') then
+        call writdx(0, 0, 0)
+     endif
+  elseif (sub_name .eq. 'writlc') then
+     if (test_case .eq. '1') then
+        call writlc(0, 0, 0, 0)
+     endif
+  elseif (sub_name .eq. 'writsa') then
+     if (test_case .eq. '1') then
+        call writsa(0, 0, 0, 0)
+     endif
+  elseif (sub_name .eq. 'writsb') then
+     if (test_case .eq. '1') then
+        call writsb(0, 0, 0, 0)
+     endif
```

Comment:
writsb has only one call argument, not 4.  Or is that the part of what you're testing here - to see how it behaves with an incorrect number of call arguments?

Reply:
No I'm trying to get the arguments correct type and number, but provide a value that causes a bort. I will fix this.